### DISCUSSION TITLE
ImportError: cannot import name 'ArraySliceDep' from 'dask.layers' when running Notebook

### DISCUSSION THREAD

**Discussion by roadbusta** [EY-Data-Science-Program/2022-Better-Working-World-Data-Challenge#4] [Q&A] [answered]
Hi all,

When trying to run the notebook, I get this error:

`ImportError: cannot import name 'ArraySliceDep' from 'dask.layers' (/anaconda/envs/2022DSC/lib/python3.8/site-packages/dask/layers.py)`

I'm not quite sure what's going on.

I'm running this on the Azure VM, and I cloned the repo from GitHub.

Any pointers?

**Comment by BillyDodds** [comment, marked as answer]
Hi @roadbusta . This might be a package dependency issue (like most of the issues in this challenge derive from). Maybe have a look at [this discussion post](https://github.com/EY-Data-Science-Program/2022-Better-Working-World-Data-Challenge/discussions/5#discussion-4050755) and try reinstalling some of the packages with the versions specified in the requirements.txt and environment.yml files. Specifically, dask. 

Hopefully that helps, let me know how you go.

Cheers, 
Billy

**Reply by roadbusta** [reply]
Thanks Billy. I followed the directions on your discussion posts and did this:

`conda create -n 2022DSC --file <folder_path>/notebooks/requirements.txt`

and it runs now. Cheers!
